dm/master/sql-operator/operator.go

```diff
@@ -132,7 +135,7 @@ func (h *Holder) Get(taskName string, sqls []string) (string, *Operator) {
 	for _, sql := range sqls {
 		for key, oper := range operators {
 			if oper.matchPattern(sql) { // matched one SQL of all is enough
-				log.Infof("[sql-operator] get an operator %s with key %s matched SQL %s", oper, key, sql)
+				h.logger.Info("get a matched operator", zap.Stringer("operation", oper), zap.String("key", key), zap.String("sql", sql))
```

Comment:
`get a matched operation`? If so, do it for L161.

Reply:
lol, unify with variable name